class-pinterest-for-woocommerce.php

```diff
@@ -295,6 +295,31 @@ public function init_plugin() {
 
 		}
 
+		/**
+		 * Initialise Tracker and add trackers to it.
+		 *
+		 * @since x.x.x
+		 *
+		 * @return Pinterest\Tracking|false
+		 */
+		public static function init_tracking() {
+			$is_tracking_enabled             = ! apply_filters( 'woocommerce_pinterest_disable_tracking', false );
```

Comment:
What about using `$is_tracking_disabled` instead of `enabled`?

You don't need to use negation here and you don't need to use negation where you check for early return.
The same question applies to the other values.